lib/rbi/parser.rb

```diff
@@ -242,10 +242,10 @@ def visit_def_node(node)
         @last_node = node
         current_scope << Method.new(
           node.name.to_s,
+          comments: current_sigs_comments + node_comments(node),
```

Comment:
I'm confused why moving this kwarg would make a difference? But maybe I'm missing something?